big_tests/tests/metrics_api_SUITE.erl

```diff
@@ -220,6 +222,16 @@ node_uptime(Config) ->
       Y = fetch_global_incrementing_gauge_value(nodeUpTime, Config),
       ?assert_equal_extra(true, Y > X, [{counter, nodeUpTime}, {first, X}, {second, Y}]).
 
+tcp_connections_increases_with_user_connection(Config) ->
+    InitialTcpCount = fetch_global_incrementing_gauge_value(tcpCountConnections, Config),
+    escalus:story(Config, [{alice, 1}],
+                 fun(_Alice) ->
+                         NewTcpCount =
+                         fetch_global_incrementing_gauge_value(tcpCountConnections, Config),
```

Comment:
this would fail sometimes, because there can be some other connections from previous tests, that are about to close.

Reply:
It shouldn't, we can't run this test in parallel, and that's why tests are ran sequentially, and `escalus:story` closes everything once finished (right?).

Reply:
Maybe or maybe not. Let's see in practice. We'll do something about it if it indeed fails randomly.